src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseCommentAttributes.java

```diff
@@ -176,7 +176,11 @@ public FeedbackResponseComment toEntity() {
 
     @Override
     public String getIdentificationString() {
-        return toString();
+        return "IdentificationString ["
```

Comment:
You don't need to make it so informative. Take a look at how other method does. `getIdentificationString` is usually used as a map key.